craftmaster.ini

```diff
@@ -42,6 +42,9 @@ QtSDK/Compiler = msvc2022_64
 General/ABI = windows-msvc2022_64-cl
 Paths/Python = C:\Python312-x64
 
+Packager/RepositoryUrl = https://download.nextcloud.com/desktop/development/craftCache/
+Packager/CacheVersion = 25.03-nc
+
```

Comment:
at this point we can move these (incl. the macOS ones) into the `GeneralSettings` section